rockspecs/lua-zlib-0.3-0.rockspec

```diff
@@ -0,0 +1,31 @@
+package = "lua-zlib"
```

Comment:
Isn't there a better way to do this that doesn't involve keeping around all these old rockspec files?

Reply:
Not that I know of. LuaRocks is kind of a pain in this regard. Having to change file names when the internal version changes does not play well with source control, but this is both a common hack and the most functional one I've seen as far as covering use cases without breaking Git.